.github/workflows/metadata_validate_manifest.yml

```diff
@@ -0,0 +1,21 @@
+name: Metadata Service CI
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
So I was leaning towards doing that.

But then I remembered the conversation we had with Conor on Monday.

I had thought we were leaning towards having all "shouldnt run" "should run" logic inside the Dagger Pipeline / CLI code?

----

Though either way, I think this is a good time to revisit that decision.

I think we may want to leave the "should / should not run logic" in github actions as long as we can to avoid reinventing the wheel.

Reply:
I agree with what we decided with Conor on Monday, but the "no filter" implementation is legit when we'll have a single CI workflow that will run multiples pipelines. As this workflow remains connector metadata specific I think it makes sense to add the filters.